optimade/validator/github_action/entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+# Retrieve and add GitHub Actions host runner IP to known hosts
+DOCKER_HOST_IP=$(cat /docker_host_ip)
+echo $DOCKER_HOST_IP gh_actions_host >> /etc/hosts
+
+run_validator="optimade_validator"
+
+if [ ! -z "$INPUT_PORT" ]; then
+    BASE_URL="$INPUT_PROTOCOL://$INPUT_DOMAIN:$INPUT_PORT"
+else
+    BASE_URL="$INPUT_PROTOCOL://$INPUT_DOMAIN"
```

Comment:
This behaviour does not seem to be working for some reason, I am leaving `port` blank in my workflow (see ml-evs/odbx.science#5), and it is defaulting to `5000`. Maybe this is a problem with the validator itself.

Reply:
#b3d3ff6dd14dd77ff108630d4548501d4eecbe8e has fixed this for me, are you happy to keep the change @CasperWA?